parlai/agents/hugging_face/dict.py

```diff
@@ -204,6 +204,9 @@ def override_special_tokens(self, opt):
         self.ind2tok[self.start_idx] = self.start_token
         self.ind2tok[self.null_idx] = self.null_token
 
+        self.unk_token = self.hf_tokenizer.unk_token
```

Comment:
Kurt has this fix in an earlier pull request https://github.com/facebookresearch/ParlAI/pull/5016/files